frontend/src/components/SupportedAppTitle.tsx

```diff
@@ -10,18 +10,34 @@ type SupportedAppTitleProps = {
 };
 
 const SupportedAppTitle: React.FC<SupportedAppTitleProps> = ({ odhApp, showProvider = false }) => {
+  const { title, icon } = React.useMemo(() => {
+    const split = odhApp.spec.displayName?.split(' ');
+
+    if (isRedHatSupported(odhApp)) {
+      return {
+        title: `${split.slice(0, -1).join(' ')} `,
+        icon: (
+          <span style={{ whiteSpace: 'nowrap' }}>
+            <span style={{ verticalAlign: 'text-bottom' }}>{split.pop()}</span>
+            <Tooltip removeFindDomNode content={`${ODH_PRODUCT_NAME} certified and supported`}>
+              <img
+                style={{ marginLeft: 'var(--pf-global--spacer--xs)' }}
+                src="../images/CheckStar.svg"
+                alt={`${ODH_PRODUCT_NAME} certified and supported`}
+              />
+            </Tooltip>
+          </span>
+        ),
+      };
+    } else {
+      return { title: odhApp.spec.displayName };
```

Comment:
You don't have to do this -- this was a note for myself. If our linter doesn't stop you, it's not really a requirement to conform.

---

Hmmm -- our linter should stop this style of code 🤔 I'll have to add the case to https://github.com/opendatahub-io/odh-dashboard/issues/469

if-based returns should not be followed up with an else return because it structures the code in a way to return a non-standard response. https://eslint.org/docs/latest/rules/no-else-return the rule I am mentioning.

Reply:
Didn't know that. I always thought it was the other way around. Will make the change

Reply:
Consider mutations to the code -- if you add another else-if case, you can easily skip the return and the function no longer has a uniform ending. Hopefully TS will catch this -- but you do not have a return type specified on this memo function, so TS may type it as `any` -- not sure off the top of my head.